docs/history.md

```diff
@@ -9,22 +9,22 @@ This page has three tabs: **Log Sheet**, **File Summary** and **Show Summary**.
 
 In the **Log Sheet** tab, the playout history is sorted by **Start Time** and **End Time** by default.
 
-<img src="static/Screenshot533-Playout_history_250.png" width="595" height="385" />
+<img src="img/Screenshot533-Playout_history_250.png" width="595" height="385" />
```

Comment:
Should these be changed to pure markdown links to be consistent with everything else?

Reply:
Do you have any comment on this?

Reply:
Not sure. That came in with the previous docs. I've fixed it.